# Need to pass in another parameter from the Makefile.

# Disable this download of outside documentation.

--- junit-r4.11/build.xml_orig	2012-11-13 12:10:08.000000000 -0800
+++ junit-r4.11/build.xml	2015-03-09 15:16:26.620136857 -0700
@@ -144,6 +144,7 @@
              version="false"
              use="false"
              windowtitle="JUnit API"
+	     additionalparam="${additionalparam}"
              stylesheetfile="stylesheet.css"
              >
       <excludepackage name="junit.*" />
@@ -152,7 +153,6 @@
       
       <sourcepath location="${src}" />
       <sourcepath location="${hamcrestsrc}" />
-      <link href="http://java.sun.com/javase/6/docs/api/" />
     </javadoc>
   </target>
 
